postgresql_watcher/watcher.py

```diff
@@ -130,29 +157,37 @@ def update(self):
             sslmode=self.sslmode,
             sslrootcert=self.sslrootcert,
             sslcert=self.sslcert,
-            sslkey=self.sslkey
+            sslkey=self.sslkey,
         )
         # Can only receive notifications when not in transaction, set this for easier usage
         conn.set_isolation_level(extensions.ISOLATION_LEVEL_AUTOCOMMIT)
         curs = conn.cursor()
-        curs.execute(
-            f"NOTIFY {self.channel_name},'casbin policy update at {time.time()}'"
-        )
+        curs.execute(f"NOTIFY {self.channel_name},'casbin policy update at {time()}'")
         conn.close()
-        return True
 
-    def should_reload(self):
+    def should_reload(self) -> bool:
```

Comment:
We can optimise should_reload function, currently if parent have 5 messages it will call update_callback on first 5 calls which can be reduces to one if we read all messages at once 

    def should_reload(self) -> bool:
        try:
            to_update_flag = False
            while self.parent_conn.poll():
                message = int(self.parent_conn.recv())
                received_update = message == _ChannelSubscriptionMessage.RECEIVED_UPDATE
                if received_update:
                    to_update_flag = True
            if to_update_flag and self.update_callback is not None:
                self.update_callback()
                return received_update
            
        except EOFError:
            self.logger.warning(
                "Child casbin-watcher subscribe process has stopped, "
                "attempting to recreate the process in 10 seconds..."
            )
            self._create_subscription_process(delay=10)

        return False

Reply:
Implemented.